west.yml

```diff
@@ -50,7 +50,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 44ede1e459c061fb1f7b65431535d6b66654ec78
+      revision: c637477892afef5e2186fe63463703aaab9c53b3
```

Comment:
You should use `pull/910/head` at this point.

Reply:
done